docs/modules/migrate/pages/lts.adoc

```diff
@@ -0,0 +1,121 @@
+= Long-term Support Releases
+
+:description: Hazelcast simplifies your upgrade experience with the introduction of long-term support (LTS) releases. You can upgrade directly from a supported previous release to the LTS release or directly between LTS releases using a rolling upgrade.
+:page-enterprise: true
+
+{description}
+
+LTS releases have been introduced with 5.5. You can upgrade directly from 5.2 or later to 5.5 without the need to upgrade to each release between your current version and the LTS release.
+
+NOTE: Any features that are marked as Beta in your current version, or any intermediate version, are not upgraded. This is due to changes that can occur between the Beta and general availability releases. If a Beta feature has been updated or made generally available in the LTS releae, that feature is included in the LTS release. For further information on rolling upgrade compatibilities, see xref:maintain-cluster:rolling-upgrades.adoc#hazelcast-members-compatibility-guarantees[Compatibility Guarantees].
+
+Between LTS releases, Hazelcast will provide short-term support (STS) releases. You can choose to upgrade directly from one LTS to the next LTS, or you can upgrade from one LTS to each STS release and then from the last STS release to the next LTS release. Before deciding the approach that best suits your needs, consider the following:
+
+[cols="1,1"]
+|===
+|LTS|STS
+
+|Focus on stability
+|Focus on innovation
+
+|Consolidates the functionality introduced in all STS releases since the last LTS release. Can include improvements, such as increased capacity limits
+|Provides the latest features and bug fixes
+
+|Released every two years
+|Released at least twice each year
+
+|Upgrade directly from the previous LTS, or from the last available STS release
```

Comment:
```suggestion
|Upgrade directly from the previous LTS, or from the last available STS before the LTS release
```

If we don't add this, is this ambiguous? for example, last STS is 5.7 and last LTS is 5.5, then is "upgrade" (well actually downgrade :D) from 5.7 -> 5.5 possible? Probably, I am overthinking it.